Dockerfile

```diff
@@ -33,12 +33,14 @@ RUN curl -OLsS https://github.com/google/protobuf/releases/download/v3.5.1/proto
  && unzip protoc-3.5.1-linux-x86_64.zip -d protoc3 \
  && rm protoc-3.5.1-linux-x86_64.zip
 
+ENV PATH=$PATH:/protoc3/bin:/project/sawtooth-raft/bin:/root/.cargo/bin \
+    CARGO_INCREMENTAL=0
+
 RUN curl https://sh.rustup.rs -sSf > /usr/bin/rustup-init \
  && chmod +x /usr/bin/rustup-init \
- && rustup-init -y
-
-ENV PATH=$PATH:/protoc3/bin:/project/sawtooth-core/bin:/root/.cargo/bin \
-    CARGO_INCREMENTAL=0
+ && rustup-init -y \
+ && rustup component add rustfmt-preview \
```

Comment:
Can we use just 'rustfmt' and 'clippy' here instead of the *-preview versions? Historically we needed the preview versions, but it probably is no longer necessary.

Reply:
Ok